File: omninavi/src/main/java/com/omni/omninavi/omninavi/model/google_navigation/GoogleDistance.java
package com.omni.omninavi.omninavi.model.google_navigation;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 * Created by wiliiamwang on 14/06/2017.
 */

public class GoogleDistance implements Serializable {

    @SerializedName("text")
    private String text;
    @SerializedName("value")
    private int value;

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

    public int getValue() {
        return value;
    }

    public void setValue(int value) {
        this.value = value;
    }
}

File: navisdk/src/main/java/com/omni/navisdk/module/google_navigation/GoogleDistance.kt
package com.omni.navisdk.module.google_navigation

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class GoogleDistance : Serializable {
    @SerializedName("text")
    var text: String? = null

    @SerializedName("value")
    var value = 0

}